### PR TITLE
Fixed the exit exception for the /url path

### LINKED ISSUE #18651: PhpMyAdmin\Exceptions\ExitException
### Describe the bug

```
Internal error in .\libraries\classes\ResponseRenderer.php#426
PhpMyAdmin\Exceptions\ExitException: 
```

### To Reproduce

1. Go to https://demo.phpmyadmin.net/master-config/public/index.php?route=/url&url=https%3A%2F%2Fwww.phpmyadmin.net%2Fsupport%2F
2. See error

### Expected behavior

No error should happen and redirect should work.

### Server configuration

- phpMyAdmin version: 6.0.0-dev

### PR DESCRIPTION
**Pull request by SachinBahukhandi**
### Description

Fixes: https://github.com/phpmyadmin/phpmyadmin/issues/18651

### CODE REVIEW

File: src/UrlRedirector.php [hunk 1]
```diff
@@ -45,6 +46,6 @@ public static function redirect(string $url): never
         // Do not display the value of $_GET['url'] to avoid showing injected content
         echo __('Taking you to the target site.');
 
-        $response->callExit();
+        return $response->response();
```
Comment (MauricioFauth) [inline src/UrlRedirector.php:49]: There is one more callExit in this file.

**Review by MauricioFauth — CHANGES_REQUESTED** [review]
You can run `vendor/bin/phpcs` to see the coding standard errors and run `vendor/bin/phpcbf` to fix them.

File: src/UrlRedirector.php (hunk 1 above)
Comment (SachinBahukhandi) [reply src/UrlRedirector.php:49]: @MauricioFauth I have made the changes. Kindly review.

**Comment by SachinBahukhandi** [comment]
> return UrlRedirector::redirect($this->getUrlParam($request->getQueryParams()['url'] ?? null));

@liviuconcioiu 
Thanks for pointing out. I have done the changes.

**Review by MauricioFauth — APPROVED** [review]

**Pull request merged**